variables.tf

```diff
@@ -131,3 +131,34 @@ variable "instance_refresh" {
 locals {
   namespace = "sp5ft-${var.worker_pool_id}"
 }
+
+variable "autoscaler_version" {
+  type        = string
+  description = "Version of the autoscaler to deploy"
+  default     = "0.1.0"
+}
+
+variable "spacelift_api_key_id" {
+  type        = string
+  description = "ID of the Spacelift API key to use"
+  default     = "0VOYU49U485BMZZVAWXU59VOW2"
```

Comment:
That should not be a default value for the variable but should be passed by the examples to the module.